cargo-dist/src/tasks.rs

```diff
@@ -613,10 +613,12 @@ pub struct Release {
     pub app_authors: Vec<String>,
     /// The license of the app
     pub app_license: Option<String>,
-    /// The license of the app
+    /// The URL to the app's source repository
     pub app_repository_url: Option<String>,
-    /// The license of the app
+    /// The URL to the app's homepage
     pub app_homepage_url: Option<String>,
+    /// A list of the app's keywords
+    pub app_keywords: Option<Vec<String>>,
```

Comment:
I don't even think adding this to the `DistGraph` was necessary, but I figured it'd be good future-proofing in case we mess around with stuff outside of npm

Reply:
Yep this is perfect.